stubs/pexpect/pexpect/fdpexpect.pyi

```diff
@@ -1,32 +1,35 @@
-from _typeshed import Incomplete
+from collections.abc import Iterable
+from typing import AnyStr
 
 from .spawnbase import SpawnBase, _Logfile
 
-class fdspawn(SpawnBase):
-    args: Incomplete
-    command: Incomplete
-    child_fd: Incomplete
+__all__ = ["fdspawn"]
+
+class fdspawn(SpawnBase[AnyStr]):
+    args: None
+    command: None
+    child_fd: int
     own_fd: bool
     closed: bool
-    name: Incomplete
-    use_poll: Incomplete
+    name: str
+    use_poll: bool
     def __init__(
         self,
-        fd,
-        args: Incomplete | None = None,
-        timeout: int = 30,
+        fd: int,
```

Comment:
Hi @tapple 

I was about to do something similar, you beat me to the punch :) (and you went further than I did).

`fd` should be [`FileDescriptorLike`](https://github.com/python/typeshed/blob/3dbb2573e68a288ac5ad1b4967a9c5ab994b81be/stdlib/_typeshed/__init__.pyi#L236), as the constructor accepts any object with a `fileno()` method (in addition to int).

Reply:
fixed